### PR TITLE
feat: semantic cache UI

### PR DESCRIPTION
**Pull request by domeccleston**

### CODE REVIEW

File: apps/dashboard/app/(app)/apis/[apiId]/page.tsx [hunk 1]
```diff
@@ -190,13 +190,7 @@ export default async function ApiPage(props: {
       )}
 
       <Separator className="my-8" />
-      <div className="flex items-center justify-between">
```
Comment (chronark) [inline apps/dashboard/app/(app)/apis/[apiId]/page.tsx:193]: why do we delete this? looks like an accident

File: apps/dashboard/app/(app)/desktop-sidebar.tsx [hunk 2]
```diff
@@ -119,7 +119,6 @@ export const DesktopSidebar: React.FC<Props> = ({ workspace, className }) => {
       href: "/semantic-cache",
       label: "Semantic Cache",
       active: segments.at(0) === "semantic-cache",
-      hidden: true,
```
Comment (chronark) [inline apps/dashboard/app/(app)/desktop-sidebar.tsx:122]: please hide again until we are ready, you could use `hidden: process.env.NODE_ENV === "production"`

File: apps/dashboard/app/(app)/mobile-sidebar.tsx [hunk 3]
```diff
@@ -52,6 +52,12 @@ export const MobileSideBar = ({ className }: Props) => {
                   Docs
                 </Button>
               </Link>
+              <Link href="https://unkey.dev/semantic-cache">
```
Comment (chronark) [inline apps/dashboard/app/(app)/mobile-sidebar.tsx:55]: please hide until we're ready

File: apps/dashboard/app/(app)/semantic-cache/page.tsx [hunk 4]
```diff
@@ -1,55 +1,32 @@
...
-  TableHeader,
-  TableRow,
-} from "@/components/ui/table";
+export default async function SemanticCachePage() {
+  const tenantId = getTenantId();
+  const workspace = await db.query.workspaces.findFirst({
+    where: (table, { and, eq, isNull }) =>
+      and(eq(table.tenantId, tenantId), isNull(table.deletedAt)),
+    with: {
+      gateways: {
```
Comment (chronark) [inline apps/dashboard/app/(app)/semantic-cache/page.tsx:13]: needs to load llmGateway

File: apps/dashboard/components/ui/tabs.tsx
Comment (chronark) [inline apps/dashboard/components/ui/tabs.tsx:1]: what's this for?
don't we have tabs already?

the used classnames are not going to work, as we don't have a "muted-bg" for example

File: apps/semantic-cache/biome.json
Comment (chronark) [inline apps/semantic-cache/biome.json:1]: if we remove this file, it should use the root level config, no?

File: apps/semantic-cache/tsup.config.js
Comment (chronark) [inline apps/semantic-cache/tsup.config.js:1]: this doesn't need a tsup config

File: apps/semantic-cache/wrangler.toml [hunk 5]
```diff
@@ -1,17 +1,23 @@
 name = "llmcache-preview"
 compatibility_date = "2023-12-01"
 
+routes = [
+  { pattern = "*.llm.unkey.io/*", zone_id = "6d4f6b5486c58f1211189367d1712db1" },
+  { pattern = "*.llm-unkey.com/*", zone_id = "a7d6ca8aa83073bef11e40a35e97af78" },
```
Comment (chronark) [inline apps/semantic-cache/wrangler.toml:6]:
```suggestion
```

File: internal/db/src/schema/index.ts [hunk 6]
```diff
@@ -9,4 +9,4 @@ export * from "./secrets";
 export * from "./gateway";
 export * from "./webhooks";
 export * from "./monitor_verifications";
-export * from "./key_migrations";
```
Comment (chronark) [inline internal/db/src/schema/index.ts:12]: why?

File: steps.txt
Comment (chronark) [inline steps.txt:1]: accidentally commited?

**Review by chronark — CHANGES_REQUESTED** [review]

File: apps/dashboard/app/(app)/desktop-sidebar.tsx (hunk 2 above)
Comment (domeccleston) [reply apps/dashboard/app/(app)/desktop-sidebar.tsx:122]: done

File: apps/dashboard/app/(app)/mobile-sidebar.tsx (hunk 3 above)
Comment (domeccleston) [reply apps/dashboard/app/(app)/mobile-sidebar.tsx]: done

File: apps/dashboard/components/ui/tabs.tsx
Comment (domeccleston) [reply apps/dashboard/components/ui/tabs.tsx:1]: removed

File: apps/semantic-cache/biome.json
Comment (domeccleston) [reply apps/semantic-cache/biome.json:1]: removed

File: apps/semantic-cache/tsup.config.js
Comment (domeccleston) [reply apps/semantic-cache/tsup.config.js:1]: removed

File: apps/semantic-cache/wrangler.toml (hunk 5 above)
Comment (domeccleston) [reply apps/semantic-cache/wrangler.toml]: removed

**Review by chronark — APPROVED** [review]

**Review by chronark — APPROVED** [review]

**Review by chronark — APPROVED** [review]

**Pull request merged**